File: Service/FinCare/RMA.Service.FinCare.Contracts/Entities/Commissions/CommissionPaymentInstruction.cs
using RMA.Common.Entities;
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.FinCare.Contracts.Entities.Commissions
{
    public class CommissionPaymentInstruction : AuditDetails
    {
        public int PaymentInstructionId { get; set; }
        public int HeaderId { get; set; }
        public CommissionPaymentTypeEnum CommissionPaymentType { get; set; }
        public decimal Amount { get; set; }
        public CommissionStatusEnum Status { get; set; }
        public string Results { get; set; }
    }
}

File: Web/RMA-Web-Shell/projects/fincare/src/app/payment-manager/models/commission-payment-instruction.ts
import { CommissionPaymentTypeEnum } from '../../shared/enum/commission-payment-type.enum';
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';
import { HeaderStatusEnum } from '../../shared/enum/header-status.enum';

export class CommissionPaymentInstruction extends BaseClass {
    paymentInstructionId: number;
    headerId: number;
    commissionPaymentType: CommissionPaymentTypeEnum;
    amount: number;
    status: HeaderStatusEnum;
    results: string; 
}
